lib/Response.php

```diff
@@ -57,6 +61,18 @@ public function __construct(
         }
     }
 
+    public function __destruct() {
```

Comment:
I think this should be final, so child classes can't remove it. If they need their own destruct, they can just register a handler.